.gitignore

```diff
@@ -18,6 +18,7 @@ app/src/androidTest/java/com/mapbox/maps/testapp/activity
 sdk/src/androidTest/res/values/developer-config.xml
 extension-style-app/src/main/res/values/developer-config.xml
 android-auto-app/src/main/res/values/developer-config.xml
+compose-app/src/main/res/values/developer-config.xml
```

Comment:
nit:
maybe refactor all lines with one `developer-config.xml` in any subfolder?